src/main/java/core/basesyntax/DataRecord.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class DataRecord {
+    private int supply;
+    private int buy;
+
+    public String getStringSupply() {
+        return String.valueOf(supply);
+    }
+
+    public String getStringBuy() {
+        return String.valueOf(buy);
+    }
```

Comment:
It seems unnecessary to have methods for getting 'supply' and 'buy' as strings. If these methods are only used to convert the integers to strings for reporting or logging, consider doing the conversion at the point where it's needed instead of having separate methods for this purpose.